scripts/scil_dilate_labels.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import argparse
+import logging
+
+import nibabel as nib
+import numpy as np
+from scipy.spatial.ckdtree import cKDTree
+
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_outputs_exist)
+
+
+DESCRIPTION = """
+    Dilate regions (with or without masking) from a labeled volume:
+    - "label_to_dilate" are regions that will dilate over
+        "label_to_fill" if close enough to it ("distance").
+    - "label_to_dilate", by default (None) will be all
+         non-"label_to_fill" and non-"label_not_to_dilate".
+    - "label_not_to_dilate" will not be changed, but will not dilate.
+    - "mask" is where the dilation is allowed (constrained)
+        in addition to "background_label" (logical AND)
+    """
+
+EPILOG = """
+    References:
+        [1] Al-Sharif N.B., St-Onge E., Vogel J.W., Theaud G.,
+            Evans A.C. and Descoteaux M. OHBM 2019.
+            Surface integration for connectome analysis in age prediction.
+    """
+
+
+def _build_args_parser():
+    p = argparse.ArgumentParser(description=DESCRIPTION, epilog=EPILOG,
+                                formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('in_file',
+                   help='Path of the volume (nii or nii.gz).')
+
+    p.add_argument('out_file',
+                   help='Output filename of the dilated labels.')
+
+    p.add_argument('--distance', type=float, default=2.0,
+                   help='Maximal distance to dilated (in mm).')
+
+    p.add_argument('--label_to_dilate', type=int, nargs='+', default=None,
+                   help='Label list to dilate, [%(default)s],\n'
+                        ' by default "None" dilate all non-background.')
```

Comment:
Please remove by default "None" because by default it is "all". 
I tried --label_to_dilate all before realizing that I don't need to use anything if I want to dilate all